plugins/official/current-timestamp/index.ts

```diff
@@ -0,0 +1,19 @@
+import "https://deno.land/std@0.188.0/dotenv/load.ts";
+import { format } from "https://deno.land/x/date_fns@v2.15.0/index.js";
+
+
+export default async (request: Request) : Promise<Response> => {
+  const json = await request.json();
+  const metadata: string = json['metadata'];
+  const note = json['note'];
+  if (!note) {
+    return new Response("Couldn't find note in request", { status: 400 });
+  }
+  const formatString = metadata ? metadata : "yyyy-MM-dd HH:mm:ss";
+  try {
+    note.content = `${note.content} ${format(new Date(), formatString, { timeZone: 'UTC' })}`;
```

Comment:
```suggestion
    const dateFormatted = format(new Date(), formatString, { timeZone: 'UTC' });
    return new Response(dateFormatted);
```

Reply:
This is just a suggestion, please test it out and see if it works

Reply:
there no need to concatenate with note?

Reply:
It'll automatically do that when you send only a string

Reply:
220a6335f7390c1fd917d93c0d4fb4d07ebcf2fe